charts/helmet/templates/_deployment.yaml

```
@@ -121,7 +121,7 @@ spec:
         {{- include "common.tplvalues.render" ( dict "value" .Values.extraVolumes "context" $) | nindent 8 }}
         {{- end }}
       {{- end }}
-      {{- include "common.images.pullSecrets" (dict "images" (list .Values.image) "global" .Values.global) | indent 6 }}
+      {{- include "common.images.renderPullSecrets" (dict "images" (list .Values.image) "global" .Values.global) | indent 6 }}
```

@zevisert Thanks for contribution, Based on the documentation https://github.com/bitnami/charts/blob/main/bitnami/common/templates/_images.tpl#L69 we have to pass `context` instead of `.Values.global`:
  ```suggestion
      {{- include "common.images.renderPullSecrets" (dict "images" (list .Values.image) "context" $) | indent 6 }}
```

---

Oh great catch! Thank you! Your suggestion removed two leading spaces, was that also intentional? I think this is the correct patch: 

```suggestion
      {{- include "common.images.renderPullSecrets" (dict "images" (list .Values.image) "context" $) | indent 6 }}
```

---

Ahh, that is my mistake.